eosio_build.sh

```diff
@@ -52,7 +52,7 @@
 	DISK_MIN=20
 	DOXYGEN=false
 	ENABLE_COVERAGE_TESTING=false
-	CORE_SYMBOL_NAME="SYS"
+	CORE_SYMBOL_NAME="EOS"
```

Comment:
Looks like this was changed by mistake.

Reply:
Didn't mean to push this here, reverting.